src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -116,7 +116,7 @@ See the [Example](plan-your-cluster.html?filters=dedicated#dedicated-example) fo
 
 ## Step 6. Name the cluster
 
-The cluster is automatically given a randomly-generated name. If desired, change the cluster's name. The cluster name must be 6-20 characters in length, and can include lowercase letters, numbers, and dashes (but no leading or trailing dashes).
+The cluster is automatically given a randomly-generated name. If desired, change the cluster's name. The cluster name must be 6-20 characters in length, and can include lowercase letters, numbers, and dashes (but no leading or trailing dashes). You cannot edit a cluster's name after it is created.
```

Comment:
```suggestion
The cluster is automatically given a randomly-generated name. If desired, change the cluster's name. The cluster name must be 6-20 characters in length, and can include lowercase letters, numbers, and dashes (but no leading or trailing dashes). A cluster's name cannot be edited after it is created.
```
To avoid the reader thinking someone else might be able to change it.

Reply:
ty!